### PR TITLE
feat: prompt environment in make:preload file

### PR DESCRIPTION
**Pull request by Julien-R44**
- Add a prompt in `node ace make:prldfile` for selecting environment 
- Also added a flag for that

### CODE REVIEW

File: commands/make/prldfile.ts [hunk 1]
```diff
@@ -8,7 +8,10 @@
  */
 
 import BaseCommand from './_base.js'
-import { args } from '../../modules/ace/main.js'
+import { args, flags } from '../../modules/ace/main.js'
+import { AppEnvironments } from '@adonisjs/application/types'
```
Comment (RomainLanz) [inline commands/make/prldfile.ts:12]:
```suggestion
import type { AppEnvironments } from '@adonisjs/application/types'
```

File: commands/make/prldfile.ts [hunk 2]
```diff
@@ -20,12 +23,79 @@ export default class MakePreloadFile extends BaseCommand {
   @args.string({ description: 'Name of the preload file' })
   declare name: string
 
+  @flags.array({
+    description: `Define the preload file environment. Accepted values "${ALLOWED_ENVIRONMENTS}"`,
```
Comment (RomainLanz) [inline commands/make/prldfile.ts:27]:
```suggestion
    description: `Define the preload file's environment. Accepted values are "${ALLOWED_ENVIRONMENTS}"`,
```

**Review by RomainLanz — CHANGES_REQUESTED** [review]

File: commands/make/prldfile.ts (hunk 1 above)
Comment (Julien-R44) [reply commands/make/prldfile.ts:12]: Definitely need that eslint rule ahah. Thanks dude

**Comment by targos** [comment]
Could we take the opportunity of a major version to rename the command so it contains "preload" instead of "prld"? This is so weird and I have never seen it abbreviated like that anywhere else.

**Comment by Julien-R44** [comment]
agree with that. i am always struggling to write it in abbreviated form

**Comment by thetutlage** [comment]
Sure, let's rename it

**Comment by RomainLanz** [comment]
Should it be named `preload` and not `preload-file`?

**Comment by thetutlage** [comment]
Yeah, whatever sounds better to you guys

**Pull request merged**